.github/dependabot.yml

```diff
@@ -281,26 +263,36 @@ updates:
     directory: "/operations/app/terraform/vars/demo"
     schedule:
       interval: "weekly"
+    ignore:
```

Comment:
I think we should keep these enabled since according to their [lifecycle policy](https://support.hashicorp.com/hc/en-us/articles/360021185113-Support-Period-and-End-of-Life-EOL-Policy), version [2.99 released](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/CHANGELOG-v2.md) in March of 2022 and may go unsupported by April 2024.